music/src/core/compat/global.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * A module containing a metronome based on Tone.js. The timing is done
+ * using the underlying WebAudio clock, so it is accurate, and the metronome
+ * fires callbacks for every audible click, quarter and bar marks.
+ *
+ * @license
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const isBrowser = typeof global === 'undefined';
```

Comment:
extremo nit pick here but I think i would call this variable "isNode" here instead.

Reply:
Sure!